test/types/list_test.rb

```diff
@@ -41,6 +41,26 @@ class ListTest < ActiveSupport::TestCase
     assert_equal [], @list.elements
   end
 
+  test "expiring" do
+    @list = Kredis.list "mylist", expires_in: 1.second
+    @list.append(%w[ 1 2 3 ])
+
+    assert_equal %w[ 1 2 3 ], @list.elements
+
+    sleep 1.1.seconds
+
+    assert_equal [], @list.elements
+
+    @list.prepend(4)
+
+    sleep 0.6.seconds
```

Comment:
I understand the usage of active delays to test this, but maybe you can replace them with using Redis' `#ttl` method, to base the assertions on having the expected TTL is set for the key. I think it's worth avoiding active delays in tests unless there is no other option (they can be brittle and slow down the suite).

Reply:
I agree; I followed the pattern for the other expiration-related tests and since the whole suite runs in 15 seconds it didn't seem pressing. Would you like me to tackle that as part of this PR?

Reply:
I see. Yes, I think it would be good to start changing the pattern in these tests, instead of extending the existing approach further.

Reply:
I updated the unit tests to use `ttl` instead of sleep. As long as the assert comes within one-second for setting it, it seems to work ok (`ttl` has a precision of seconds) and avoids the sleeps.

I left the sleeps in the `attributes_test` as that seems more appropriate to not test the internals of the kredis types.